src/esthri/Cargo.toml

```diff
@@ -40,9 +40,10 @@ name = "esthri"
 path = "src/lib.rs"
 
 [features]
-default = [ "rustls", "cli", "http_server", "blocking",]
+default = [ "rustls", "cli", "http_server", "blocking", "compression",]
```

Comment:
Do the rust powers that be have any guidelines on the use of hyphens versus underscores for feature names? Just looking at this file it seems a few of them use either/or.

Reply:
I think I've seen both, so I don't think there's a consensus